src/main/java/io/vertx/json/schema/JsonSchemaOptions.java

```diff
@@ -40,6 +41,11 @@ public class JsonSchemaOptions {
    */
   private OutputFormat outputFormat = OutputFormat.Flag;
 
+  /**
+   * The json format validator that is used during schema validation.
+   */
+  private JsonFormatValidator jsonFormatValidator = new DefaultJsonFormatValidatorImpl();
```

Comment:
```suggestion
  private JsonFormatValidator jsonFormatValidator = (format, instance) -> null;
```

No need for `DefaultJsonFormatValidatorImpl`